documentation/routing/tutorial-routing-annotation.asciidoc

```diff
@@ -30,8 +38,12 @@ public class SomePathComponent extends Div {
 }
 ----
 
-So whenever the user navigates to `http://yourdomain.com/some/path` (assuming the app is running from the root context), either by clicking on links inside the application or by typing the address directly on the address bar, the `SomePathComponent` component would be shown at the page.
+* Assuming your app is running from the root context, when the user navigates to `http://yourdomain.com/some/path`, either by clicking a link in the application or entering the address in the address bar, the `SomePathComponent` component is shown on the page. 
+
+[NOTE]
+If you omit the `@Route` annotation parameter, the route target is derived from the class name. For example, MyEditor becomes "myeditor", PersonView becomes "person" and MainView becomes "".
+
+// I think this NOTE needs further explanation. It is not obvious to me why and how the route target is derived.
 
-TIP: If you leave out the parameter to a Route annotation, the route target will be derived from the class name. For example MyEditor will become "myeditor", PersonView will become "person" and MainView will become "".
 
-For nested route definitions see: <<tutorial-router-layout#route-prefix,ParentLayout route control using @RoutePrefix>>
+// I think this section needs additional background and introductory content. I think we should include a new page "About Routing and Navigation" that describes in general terms (high-level) what routing is and how it is used for navigation. Section 11.8 of the Vaadin 8 book contains a nice summary, but it is out of date. The content at https://vaadin.github.io/vaadin-router/vaadin-router/demo/#vaadin-router-getting-started-demos is  good, but I'm not too sure how it relates or how to include it. 
```

Comment:
Move this line to new issue in GitHub. We can then ask help form the development team to improve these parts.

Reply:
https://github.com/vaadin/flow-and-components-documentation/issues/403